Sources/OktaLogger/Analytics/Analytics.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  AnalyticsManager.swift
+//  OktaLogger
+//
+//  Created by SP on 8/9/22.
+//  Copyright © 2022 Okta, Inc. All rights reserved.
+//
+
+import Foundation
+import AppCenterAnalytics
+
+@objc public enum Provider: Int {
+    case appCenter
+}
```

Comment:
I say we drop this for now and just hardcode to AppCenter like we did with the other ones